src/SEBT.Portal.Api/Controllers/Auth/OtpController.cs

```diff
@@ -0,0 +1,69 @@
+using Microsoft.AspNetCore.Mvc;
+using SEBT.Portal.Kernel;
+using SEBT.Portal.Kernel.AspNetCore;
+using SEBT.Portal.UseCases.Auth;
+
+namespace SEBT.Portal.Api.Controllers;
```

Comment:
Is rate-limiting OTP requests part of the scope of this?  If not, it can added into [DC-31](https://codeforamerica.atlassian.net/jira/software/c/projects/DC/boards/352?selectedIssue=DC-31) maybe?

I'm just thinking of a scenario where a malicious user might attempt to spam the endpoint to force either a CAN-SPAM violation, a DDOS attempt, or something along those lines.  I could just be thinking too far ahead though.